packages/p2p/src/stores/general-store.js

```diff
@@ -760,12 +758,11 @@ export default class GeneralStore extends BaseStore {
             this.ws_subscriptions.order_list_subscription.unsubscribe();
             return;
         }
-
-        const { p2p_order_list, p2p_order_info } = order_response ?? {};
+        const { p2p_order_info } = order_response ?? {};
         const { order_store } = this.root_store;
 
-        if (p2p_order_list) {
-            const { list } = p2p_order_list;
+        if (this.p2p_order_list_response.length && this.p2p_order_list_response.length > 0) {
```

Comment:
`this.p2p_order_list_response.length && this.p2p_order_list_response.length > 0` do we need two checks? dont they logically mean the same?

Reply:
Yes I have changed the condition now